backend/volontulo_org/settings/base.py

```diff
@@ -42,8 +42,8 @@
     'bootstrap3',
     'cookielaw',
     'rest_framework',
-
     'apps.volontulo',
+    'raven.contrib.django.raven_compat'
```

Comment:
Do we want to enable Raven in debug mode? I'd say no. Maybe we'd load some apps (raven, django debug toolbar) only in debug mode?

Reply:
That is in `base.py`, not in eg. `dev_local.py`. Or am I missing something? ;-)